psi4/src/psi4/dfocc/t2_2nd_gen.cc

```diff
@@ -85,6 +85,13 @@ void DFOCC::t2_2nd_gen() {
         // Denom
         Tnew->apply_denom_chem(nfrzc, noccA, FockA);
 
+        // DIIS
+        RT2 = SharedTensor2d(new Tensor2d("RT2_2 (IA|JB)", naoccA, navirA, naoccA, navirA));
```

Comment:
Given that we just had a PR to move to `make_shared`, could we get rid of the `new` introduced by this PR?